packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -73,7 +73,7 @@ export interface LimboMap {
 
 export interface ActiveTargetSpec {
   queries: SpecQuery[];
-  resumeToken: string;
+  resumeToken: string | TestSnapshotVersion;
```

Comment:
This should probably be split into two distinct fields, since TestSnapshotVersion is not a resumeToken.

Reply:
Done.